src/Neo.VM/Unsafe.cs

```diff
@@ -38,5 +40,40 @@ public static bool NotZero(ReadOnlySpan<byte> x)
             }
             return false;
         }
+
+        /// <summary>
+        /// Get 64-bit hash code for a byte array
+        /// </summary>
+        /// <param name="span">Span</param>
+        /// <returns></returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static long HashBytes(ReadOnlySpan<byte> span)
```

Comment:
`XxHash3` may be better.
see #3543

Reply:
Agree, we can change it

Reply:
need to check and see if it faster then what we current have.

Reply:
`XxHash3` is calculated using SIMD, so it it faster than byte by byte.

Reply:
Done